apps/emqx_ds_backends/test/emqx_ds_backends_SUITE.erl

```diff
@@ -686,38 +760,39 @@ delete(DB, It0, Selector, BatchSize, Acc) ->
 %% CT callbacks
 
 all() ->
-    [{group, builtin_local}, {group, builtin_raft}].
+    [{group, Backend} || Backend <- backends()].
+
+exclude(emqx_ds_fdb_backend) ->
+    [
+        t_09_atomic_store_batch,
+        t_11_batch_preconditions,
+        t_12_batch_precondition_conflicts,
+        t_smoke_delete_next
+    ];
+exclude(_) ->
+    [].
 
 groups() ->
     TCs = emqx_common_test_helpers:all(?MODULE),
+    [{Backend, TCs -- exclude(Backend)} || Backend <- backends()].
+
+init_per_group(emqx_fdb_ds, Config) ->
+    {skip, fixme};
```

Comment:
Will be fixed downstream?

Reply:
emqx_fdb_ds has its own test suite so there's no pressure to adopt backends_suite